### PR TITLE
use minitest-reporters

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
```
/home/jenkins/workspace/test_develop_pr_core@2/database/postgresql/ruby/2.7/slave/fast/test/unit/parameter_filter_test.rb:5:in `block in <class:ParameterFilterTest>': Mocha methods cannot be used outside the context of a test (Mocha::NotInitializedError)
```

what is this?!

**Comment by ezr-ondrej** [comment]
Is the foreman failing because of `[Checks API] No suitable checks publisher found.` ?

**Comment by evgeni** [comment]
No, it's failing because 
```
test/unit/parameter_filter_test.rb:5:in `block in <class:ParameterFilterTest>': Mocha methods cannot be used outside the context of a test (Mocha::NotInitializedError)
```

Now, I have no idea why it tries to call `Mocha.mock` and not `Minitest.mock` there. But hey. MORE PROBLEMS.

File: lib/tasks/jenkins.rake [hunk 1]
```diff
@@ -11,10 +10,9 @@ begin
 
     namespace :setup do
       task :pre_ci do
-        ENV["CI_REPORTS"] = 'jenkins/reports/unit/'
-        gem 'ci_reporter'
+        ENV['USE_MEAN_TIME_REPORTER'] = '0'
```
Comment (evgeni) [inline lib/tasks/jenkins.rake:13]: this can be dropped, once https://github.com/Katello/katello/pull/9826 is merged

**Comment by evgeni** [comment]
cc @jturel as you originally implemented the Meantime Reporter in Katello

**Comment by ezr-ondrej** [comment]
:tada: shiny

File: lib/tasks/jenkins.rake (hunk 1 above)
Comment (ezr-ondrej) [reply lib/tasks/jenkins.rake:13]: Can we have a comment for that, or do we believe oursleves to remember?

Comment (evgeni) [reply lib/tasks/jenkins.rake:13]: as the author of both, I'll try to remember? ;)

**Review by ezr-ondrej — APPROVED** [review]
Thanks @evgeni ! Great effort to improve our tests! :+1:

**Pull request merged**